blueprints/getting-started/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "kubernetes_version" {
+  type        = string
+  description = "Kubernetes version to use for EKS Cluster"
+  default     = "1.21"
```

Comment:
*Nit* - Kubernetes 1.22 is out for EKS and has some nice new features! But this nit isn’t a blocker to merging this in.

Reply:
Let me run a few tests - if they come back clean, I'll bump to `1.22`